ethash/src/lib.rs

```diff
@@ -143,9 +143,11 @@ mod benchmarks {
 
 	#[bench]
 	fn bench_light_compute(b: &mut Bencher) {
+		use ::std::env;
+
 		let hash = [0xf5, 0x7e, 0x6f, 0x3a, 0xcf, 0xc0, 0xdd, 0x4b, 0x5b, 0xf2, 0xbe, 0xe4, 0x0a, 0xb3, 0x35, 0x8a, 0xa6, 0x87, 0x73, 0xa8, 0xd0, 0x9f, 0x5e, 0x59, 0x5e, 0xab, 0x55, 0x94, 0x05, 0x52, 0x7d, 0x72];
 		let nonce = 0xd7b3ac70a301a249;
-		let light = Light::new(486382);
+		let light = Light::new(env::temp_dir(), 486382);
```

Comment:
I think we do use it in several places, but you are right `env::temp_dir` is way more common. I will create a separate issue for this